frontend/controller/app/identity.js

```diff
@@ -518,7 +518,15 @@ export default (sbp('sbp/selectors/register', {
         // `.cheloniaState` key. This will be used later when logging in
         // to restore both the Vuex and Chelonia states
         if (!wipeOut) {
+          // Save current Chelonia state into the state to be saved
           state.cheloniaState = cheloniaState
+          // These keys (contract state) will be restored from the Chelonia
+          // state upon login (see `loadState`). Remove them to avoid storing
+          // unnecessary data.
+          Object.keys(cheloniaState.contracts || {}).forEach((contractID) => {
+            delete state[contractID]
+          })
+          delete state.contracts
```

Comment:
Is there a bug this is preventing? If so, could you update the comment to mention it?